Assets/Scripts/Menus/SettingsManager.cs

```diff
@@ -292,11 +292,11 @@ private void LoadSettings()
         
         try
         {
-            GameSettings.MusicVolume = PlayerPrefs.GetFloat("MusicVolume", .75f); //Default is 75% volume
+            GameSettings.MusicVolume = PlayerPrefs.GetFloat("MusicVolume", .5f); //Default is 50% volume
         }
         catch
         {
-            GameSettings.MusicVolume = .75f; //Default is 75% volume in case the PlayerPrefs value is invalid
+            GameSettings.MusicVolume = .5; //Default is 50% volume in case the PlayerPrefs value is invalid
```

Comment:
Add suffix 'f' because unity threw an error about how doubles can't be floats and such.

Reply:
Fixed